api-server/README.md

```diff
@@ -64,11 +64,10 @@ We have provided some scripts that should facilitate installation of the API ser
 bash -c "$(curl -fsSL https://raw.githubusercontent.com/instructlab/ui/refs/heads/main/api-server/rhelai-install/install-glibc-devel.sh)"
```

Comment:
@Gregory-Pereira do we still need glibc-devel installation?

Reply:
AFAIK unless you are packaging the shared-library along with the binary, yes we will need the library to run the code.

Reply:
It's devel packages, so binary should execute without it, but i will give it a try locally and see if i hit any issue without this package.